internal/pkg/process/module.go

```diff
@@ -54,6 +54,9 @@ func (a *Analyzer) getModuleDetails(f *elf.File) (*version.Version, map[string]s
 
 func parseGoVersion(vers string) (*version.Version, error) {
 	vers = strings.ReplaceAll(vers, "go", "")
+	if idx := strings.Index(vers, " X:"); idx > 0 {
```

Comment:
Please add a comment explaining what this is doing. The use of the experimental flag for Go is non-standard and should be explained here.